homeassistant/components/todoist/calendar.py

```diff
@@ -121,6 +123,8 @@ async def async_setup_platform(
     collaborator_id_lookup = {}
 
     api = TodoistAPIAsync(token)
+    coordinator = TodoistCoordinator(hass, _LOGGER, SCAN_INTERVAL, api)
+    await coordinator.async_config_entry_first_refresh()
```

Comment:
I updated this based on a comment in the [DataUpdateCoordinator](https://github.com/boralyl/home-assistant/blob/48fca3bb2727b393d29f2246bc4df98ba8c35ed6/homeassistant/helpers/update_coordinator.py#L76-L80) from the previous `coordinator.async_refresh()`.